### DISCUSSION TITLE
v2.0.0

### DISCUSSION THREAD

**Discussion by rgossiaux** [rgossiaux/svelte-headlessui#146] [Announcements]
## What's Changed
* Remove TestRenderer from Dialog and Disclosure tests by @davepar in https://github.com/rgossiaux/svelte-headlessui/pull/101
* Fix Typo "PopverPanel" in Popover by @safwanolaimat in https://github.com/rgossiaux/svelte-headlessui/pull/144
* Error when styling tabs using Docs example by @VitroidFPV in https://github.com/rgossiaux/svelte-headlessui/pull/100
* Version 2.0 by @rgossiaux in https://github.com/rgossiaux/svelte-headlessui/pull/96
  * **Breaking**: changes component API for `Listbox`, `Switch`, and `RadioGroup` to use `bind:`
  * **Breaking**: renames the events emitted by `Transition`
  * Now uses `<svelte:element>` internally

## New Contributors
* @davepar made their first contribution in https://github.com/rgossiaux/svelte-headlessui/pull/101
* @safwanolaimat made their first contribution in https://github.com/rgossiaux/svelte-headlessui/pull/144
* @VitroidFPV made their first contribution in https://github.com/rgossiaux/svelte-headlessui/pull/100

**Full Changelog**: https://github.com/rgossiaux/svelte-headlessui/compare/v1.0.2...v2.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/rgossiaux/svelte-headlessui/releases/tag/v2.0.0'>v2.0.0</a>.</em>

**Comment by rgossiaux** [comment]
I know people have been hoping for some more updates to this library since last year. I'm trying to catch up now on the backlog. My first priority is to make sure the library is compatible with Svelte 4. After that I have a long backlog of changes to port from the Tailwind library, first and foremost of which is the Combobox component.

**Reply by mfilteau** [reply]
Can I buy you a coffee?

**Reply by eliasdefaria** [reply]
The legend returns! Very excited to see some of the asks come to life. Huge fan of this port

**Reply by connerdassen** [reply]
Where have you been?

**Reply by sureshjoshi** [reply]
Woot! 

Also, FWIW, I've been using Noel's (combobox) and chaseweaver's (multi-select listbox) PRs to much success over the past few months.
